Jenkinsfile

```diff
@@ -41,8 +41,11 @@ properties([
         stringParam(defaultValue: '10.40.9.2', description: 'IP of the WAN board',
             name: 'WAN_IP'),
         stringParam(defaultValue: '/home/jenkins/boardfarm/boardfarm',
-            description: 'Path to the a local copy of Boardfarm repository',
-            name: 'BOARDFARM_DIRECTORY'),
+            description: 'Path to a local copy of OTA scripts',
+            name: 'OTA_DIRECTORY'),
+        stringParam(defaultValue: 'https://github.com/CreatorDev/boardfarm.git',
```

Comment:
I also wonder whether we can do away with all these boardfarm related values as build parameters, and instead use them as environment variables or constants?  Basically, these show up in Build parameters, which we know aren't going to be changed for the hardware setup while triggering a manual build. For e.g., we know the boardfarm repo URL is not going to change

![image](https://cloud.githubusercontent.com/assets/13044939/21672341/f4e7422e-d347-11e6-8f3b-edbbd72f23f8.png)